### DISCUSSION TITLE
无法限制实例下某个库的查询权限，archery库中实例的密码未加密

### DISCUSSION THREAD

**Discussion by wutao-git** [hhyo/Archery#2129] [Q&A]
### 重现步骤

我在业务实例下新建了archery的库，纳管该实例后，无法禁止开发人员查询archery这个库

### 预期外的结果

应该有对实例下各库的权限管理

### 版本

1.9.1

### 部署方式

K8S

**Comment by nick2wang** [comment]
去掉该实例的管理账号对archery库的权限即可

**Reply by wutao-git** [reply]
使用的是root，已经决定做迁移了，如果sql_instance的密码是加密的就无所谓了

**Reply by hhyo** [reply]
是加密的，在库里面查询可以看到

**Reply by jeromeji** [reply]
库里面是加密的，但是admin里面修改实例配置，f12可以看到没加密的密码。。。

**Reply by jeromeji** [reply]
> 



> 是加密的，在库里面查询可以看到

库里面是加密的，但是admin里面修改实例配置，f12可以看到没加密的密码。。。
